components/salsa-macro-rules/src/setup_tracked_struct.rs

```diff
@@ -256,6 +260,7 @@ macro_rules! setup_tracked_struct {
                 }
 
                 $(
+                    $(#[$tracked_field_attr])*
```

Comment:
Should we also set the attributes on the setter and builder methods. E.g. it would be useful to have the documentation everywhere.

Reply:
Idk how we can distinguish between doc comments for setter and getter then. Also it seems more natural for me to pass attributes from "field" to getter with the same name, but not to setter

Reply:
I'd be fine having the same doc comment but we can also defer this for now

Reply:
I'm completely fine with getter-only doc comments. If anyone really want attributes for setters, it wont be hard to implement, so lets leave it as is for now